src/Service/OrbitService.cpp

```diff
@@ -110,13 +110,13 @@ bool IsSshConnectionAlive(std::chrono::time_point<std::chrono::steady_clock> las
              .count() < timeout_in_seconds;
 }
 
-std::unique_ptr<OrbitGrpcServer> CreateGrpcServer(uint16_t grpc_port, bool dev_mode) {
+ErrorMessageOr<std::unique_ptr<OrbitGrpcServer>> CreateGrpcServer(uint16_t grpc_port,
+                                                                  bool dev_mode) {
   std::string grpc_address = absl::StrFormat("127.0.0.1:%d", grpc_port);
   ORBIT_LOG("Starting gRPC server at %s", grpc_address);
   std::unique_ptr<OrbitGrpcServer> grpc_server = OrbitGrpcServer::Create(grpc_address, dev_mode);
   if (grpc_server == nullptr) {
-    ORBIT_ERROR("Unable to start gRPC server");
-    return nullptr;
+    return ErrorMessage{"Unable to start gRPC server"};
```

Comment:
```suggestion
    return ErrorMessage{"Unable to start gRPC server."};
```
As it ends up in the UI, I assume as the last part of a message.